### PR TITLE
Jordan part3

### PR DESCRIPTION
**Pull request by pbxed**

### CODE REVIEW

File: src/main/java/com/codingnomads/part_03/Exercise_01.java [hunk 1]
```diff
@@ -12,3 +12,79 @@ The first class (the controller) should have at least two methods, one of which
...
+    }
+}
+
+class FootballLeagues {
+
+    private String leagueName;
+    private String leagueLocation;
+    private int numberOfTeams;
+
+    FootballLeagues(String leagueName, String leagueLocation, int numberOfTeams){
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_03/Exercise_01.java]: This constructor is taking three parameters but not assigning their data to any variables, hence getters do not work as expected - assign paramaters to class variables

File: src/main/java/com/codingnomads/part_03/Exercise_01.java [hunk 2]
```diff
@@ -12,3 +12,79 @@ The first class (the controller) should have at least two methods, one of which
   sports teams, trees, beers, people and so on.
 
  **/
+
+public class Exercise_01 {
+
+    public static void main(String[] args) {
+
+        FootballLeagues EPL = new FootballLeagues("English Premier League","England & Wales", 20);
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_03/Exercise_01.java]: Variable names that are not constants should not be all capitalized

File: src/main/java/com/codingnomads/part_03/Exercise_03.java [hunk 3]
```diff
@@ -11,7 +11,7 @@ public static void main(String[] strings) {
 
 
         // use a for loop to iterate 1000 times
-        for (int i = 0; i < 1000; i++) {
+        for (int i = 0; i < 10; i++) {
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_03/Exercise_03.java]: How come you changed it? :) it was perfect with '1000'. No big deal

File: src/main/java/com/codingnomads/part_03/Exercise_04.java [hunk 4]
```diff
@@ -53,8 +77,22 @@ public static String determineWinner(int computer, int player) {
...
 
+        if (computer == player) {
+
+            status = "You tied!";
+        }
+
+
         // if it is not a tie, use a switch statement and a turnery operator to determine the winner
 
+        switch (player) {
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_03/Exercise_04.java]: The logic might be backwards or incorrect. When I ran it I got the following results:
image

File: src/main/java/com/codingnomads/part_03/Exercise_05.java [hunk 5]
```diff
@@ -1,20 +1,42 @@
 package com.codingnomads.part_03;
+import java.util.Scanner;
 
 public class Exercise_05 {
 
     public static void main(String[] strings) throws java.io.IOException {
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_03/Exercise_05.java]: 💯

File: src/main/java/com/codingnomads/part_03/Exercise_06.java [hunk 6]
```diff
@@ -16,18 +16,60 @@ public static void main(String[] strings) {
         long totalMilliseconds = System.currentTimeMillis();
 
         // Get total seconds since midnight, 1/1/1970
+        long totalSeconds = (timeZoneChange * 60 * 60 ) + (System.currentTimeMillis()/1000);
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/part_03/Exercise_06.java]: It seems to work fine with positive timezones but not negative ones as seen below. The AM / PM thing would need to be adjusted too but I don't know how to do it either lol
image

Comment (pbxed) [reply src/main/java/com/codingnomads/part_03/Exercise_06.java]: fixed.

**Pull request merged**